CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+
+### Table of Content
+
+- [Welcome](#welcome)
+- [Code of Conduct](#code-of-conduct)
+- [Environment Details](#environment-details)
+- [How Can I Contribute?](#how-can-i-contribute)
+    - [Submitting an Issue](#submitting-an-issue)
+    - [Your First Contribution](#your-first-contribution)
+    - [Making a Pull Request](#making-a-pull-request)
+- [Styleguides](#styleguides)
+    - [Git Commit Messages](#git-commit-messages)
+    - [Git Branch Naming](#git-branch-naming)
+    - [Pull Requests](#pull-requests)
+    - [Issues](#issues)
+
+## Welcome
+
+🎉 Welcome to the Black Jack Is Fun project! We're thrilled that you're considering contributing to our repository. Before you get started, please take a moment to read through this guide to understand how you can contribute effectively and make the most out of your involvement.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[blackjack-is-fun Code of Conduct](/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. 
+
+## Environment Details
+
+This project has a variety of dependencies required to run. This section will act as a guide to setting up the environment for this project.
+
+#### Install Node.js and npm
+1. Download and install Node.js from https://nodejs.org/. 
+2. npm (Node Package Manager) is included with Node.js, so you don't need to install it separately.
+
+#### Clone Repo
+1. Clone the repository
+    >`git clone https://github.com/jacketattack/blackjack-is-fun.git`
+
+#### Install Dependencies
+1. Install dependencies: `npm i`
+2. Start app: `npm run start` and navigate to `http://localhost:1234` and beat the dealer!
+
+## How Can I Contribute?
+
+### Submitting an Issue
+
+We use github's [issue tracker](https://github.com/jacketattack/blackjack-is-fun/issues) to contain bug reports and feature/enhancement requests. Before submitting an issue, search to ensure that there are no similar issues. When raising an issue, follow the [issues styleguide](#issues).
+
+### Your First Contribution
+
+Unsure where to begin contributing to blackjack-is-fun? You can start looking through the [good first issue](https://github.com/jacketattack/blackjack-is-fun/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and [help wanted](https://github.com/jacketattack/blackjack-is-fun/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) issues. There are other labels that can be used to better find an issue that fits your needs, but these will be the most beginner-friendly.
+
+Once you find an issue you like and are assigned to it, make a separate branch following the [styleguide](#git-branch-naming) for naming git branches.
+
+### Making a Pull Request
+
+There are some rules before a pull request can be reviewed:
+1. There should only be one commit
+2. The branch should be rebased with main. Visit [git branching](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) to learn more about rebasing
```

Comment:
`We require that PRs are kept up to date with the target branch thus they should be rebased with the target branch`